nixos/modules/services/misc/gitea.nix

```diff
@@ -349,7 +349,7 @@ in
       server = mkMerge [
         {
           DOMAIN = cfg.domain;
-          STATIC_ROOT_PATH = cfg.staticRootPath;
+          STATIC_ROOT_PATH = toString cfg.staticRootPath;
```

Comment:
This used to be a `"${x}"` which is different from `toString`. The prior adds path-values to the store, whereas `toString` converts path-values to path strings without adding to the store.

Reply:
Not sure which behavior is better.

Reply:
For the example (`pkgs.gitea`) it doesn't make a difference.
For users, they should be able to decide if they want to add a path to the store or not.

Reply:
sgtm.